Misc/NEWS.d/next/Library/2017-09-13-21-25-23.bpo-28556.TnT4VI.rst

```diff
@@ -0,0 +1,2 @@
+Minor updates to the ``typing`` module.  Original PRs by Ivan Levkivskyi and
```

Comment:
Instead of "Minor updates" can you describe what the updates are?

Reply:
They are quite technical: replacing ``_gorg`` function with a pre-calculated attribute, and adding one test that verifies preserved identity after copying a generic class. Should I add this?

Reply:
Instead of saying "Minor updates", say: "Speed improvements".

Reply:
> Instead of saying "Minor updates", say: "Speed improvements".

OK, fixed.